### ISSUE TITLE
Process all of SRA

### ISSUE THREAD

**Issue by jvivian** [BD2KGenomics/toil-rnaseq#31]
- Containerize SRA toolkit
- Spot check several SRA datasets for packaging formats (get access?)
- Modify pipeline to download SRA as input
- Start Azure boxes

**Comment by jvivian** [comment]
Blocked on apparent Azure instance cap — can't create enough instances.

**Comment by jvivian** [comment]
Waiting to hear back from SRA about pulling ~180TB of compressed down efficiently.

**Comment by jvivian** [comment]
Waiting on Azure support

**Comment by diekhans** [comment]
GENCODE would find it highly desirable for the results to be submitted to the SRA/ENA as analysis on top of the original submissions.  This will assign assessions and allow provenance tracking.   This will also help increase the visibility of the project as it will probably be the single largest (in terms of number) of objects submitted.

**Comment by diekhans** [comment]
GENCODE would find it highly desirable for the results to be submitted to the SRA/ENA as analysis on top of the original submissions.  This will assign assessions and allow provenance tracking.   This will also help increase the visibility of the project as it will probably be the single largest (in terms of number) of objects submitted.

**Comment by diekhans** [comment]
Riken FANTOM5 project is interested in having a transcript assembly step included in our run.  The output would be a BED or GTF, so reasonable small.

**Comment by diekhans** [comment]
Toil should now be stable to run on Azure without any type of autoscaling.   A fixed cluster should meet the needs of this project.

It was suggested that hisat2 be used instead of STAR.  It might actually be interesting to use both and generate a dataset for someone to compare.

**Comment by diekhans** [comment]
The CZI splicing junctions were generated without poly-A trimming.   This caused a very large number of mappings to incorrect locations.   If not already done, we should look at discard reads with poly-A tails.

**Comment by diekhans** [comment]
From: Chung-Chau HON <chungchau.hon@riken.jp>
To: markd@soe.ucsc.edu
Subject: Re: SRA RNA-Seq run
Date: Tue, 3 Oct 2017 14:28:39 +0900

Hi Mark,

	Thanks for letting me know.
	I am interested to run stringtie (https://ccb.jhu.edu/software/stringtie/) on the bams of the SRA for de novo assembly using default parameters.
	The only file I want to keep is the gtf.
	The run should be straightforward but the strand options might be tricky.
	If the metadata of the SRA clearly states the strand specificity then it’s ok.
	If not, I might write a little perl script to determine the strand specificity from the bam.
	If you are planning to do some test runs, let me know.

	Are you guys also doing Mouse SRA?

cheers,
Chung

**Comment by diekhans** [comment]
From: Chung-Chau HON <chungchau.hon@riken.jp>
To: Mark Diekhans <markd@soe.ucsc.edu>
Subject: Re: SRA RNA-Seq run
Date: Tue, 3 Oct 2017 15:13:30 +0900

Hi Mark,

	The program would look into a number of chosen loci, e.g. rRNAs and house keeping genes like actin and GAPDH etc, then check the orientation of the read relative to those annotations.
	Maybe I could come up with the script first and see how well it works.

cheers,
Chung

**Comment by diekhans** [comment]
Max claims that Nuno Fonseco at EBI has mapped SRA transcriptome data to hg38/mm10.

I think this is the paper: http://europepmc.org/abstract/MED/28369191

**Comment by diekhans** [comment]
Max pointed me to the program regextract  https://github.com/griffithlab/regtools, which can find splice junctions in already aligned reads.

Not clear if we should use this or a similar tool or just rerun mapper.  It might not save any time, especially human time, using prebuilt alignments.

**Comment by diekhans** [comment]
cutadapt will trim poly-A tails in a smart way:

http://cutadapt.readthedocs.io/en/stable/recipes.html

**Comment by diekhans** [comment]
HAVANA suggests adding http://www.blueprint-epigenome.eu/

Restricted access, but easy to get

**Comment by jvivian** [comment]
Closing in lieu of other planned recomputes
